app/models/test_track/assignment.rb

```diff
@@ -16,7 +16,11 @@ def variant
   end
 
   def unsynced?
-    true
+    !feature_gate?
+  end
+
+  def feature_gate?
+    split_name.end_with?('_enabled')
```

Comment:
The wire protocol TestTrack uses for split registries is too concise to add an explicit field describing gate-iness without changing it completely. Currently it's like so:

```JSON
{
  "split_name": { "variant1": 50, "variant2": 50 }
}
```

So I'm planning on making `_enabled` a hard requirement of feature gate naming as we advance from here so that the client can reliably infer gate-iness this way.